diff --git a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
index 0fd8d9a..510ed0b 100644
--- a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
+++ b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
@@ -918,5 +918,6 @@ function parseDeclaration(declaration, options) {
         case "box-shadow": {
             parseBoxShadow(declaration.value, parseOptions);
+            return;
         }
         case "aspect-ratio": {
             return addStyleProp(declaration.property, parseAspectRatio(declaration.value));
diff --git a/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts b/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
index 080bfa5..13e1ae1 100644
--- a/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
+++ b/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
@@ -1518,5 +1518,6 @@ export function parseDeclaration(
     case "box-shadow": {
       parseBoxShadow(declaration.value, parseOptions);
+      return;
     }
     case "aspect-ratio": {
       return addStyleProp(declaration.property, parseAspectRatio(declaration.value));
